draft-ietf-rats-eat.md

```diff
@@ -1244,6 +1262,8 @@ following CDDL types are encoded in JSON as follows:
 * bstr -- must be base64url encoded
 * time -- must be encoded as NumericDate as described section 2 of {{RFC7519}}.
 * string-or-uri -- must be encoded as StringOrURI as described section 2 of {{RFC7519}}.
+* uri -- must be a URI {{RFC3986}}.
+* oid -- is encoded as a string represetation of the OID digits.
```

Comment:
```suggestion
* oid -- is encoded using dotted decimal notation as per Section 5 of {{CBOR-OID}} 
```

Reply:
This is JSON representation of OIDs so a reference to CBOR representation of OID seems wrong.
I haven't found any standard to reference for text representation of OIDs in JSON so far.

Reply:
Yeah that doesn't sound entirely right, but it's the only reference I had found.

I've given my google-foo another spin and found that Section 3.2 of RFC5675 has an ABNF for the dotted-decimal notation.  Or otherwise let's just say "using the well established dotted-decimal notation" or similar

Reply:
OK. Pushed a change for this.